contribute/creating_package.md

```diff
@@ -0,0 +1,93 @@
+# Creating a Julia package
+
+A detailed page on Julia packages is available [here](http://pkgdocs.julialang.org/v1/). Including the part on [creating packages](http://pkgdocs.julialang.org/v1/creating-packages/).
+
+Julia packages consist of modules. Hence, before creating a package, make sure to understand how [modules](https://docs.julialang.org/en/v1/manual/modules/) work in Julia.
+
+
+Here we provide a step-by-step example, using [PkgTemplates.jl](https://juliahub.com/docs/PkgTemplates/IGiQL/0.6.4/), inspired by the [video](https://www.youtube.com/watch?v=QVmU29rCjaA&t=112s) of Chris Rackauckas.
+
+In case of any issues, remember that [help](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#where-to-get-help) is available.
+
+## What you need before you start
+
+- an [IDE](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#what-are-the-ide-options)
+- a GitHub account
+
+## Starting a package
+
+- create a folder to store your package. We use Atom as an Editor. We created a folder called 'tiddlywinks.jl'. In Atom, we open the GitHub panel, and opt to 'initialize and publish on GitHub'. Now we can go to our GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
```

Comment:
This section is a bit confusing :( We could reprhase it as follows:

1. You need to create a folder to store a new package. For example, if you use Atom as an IDE, create a new folder called `tiddlywinks.jl`. Then, open the GitHub panel and select **Initialize and publish on GitHub**. 
  Now, you can go to your GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
2. In your terminal application, use the change directory (cd) command to change the current working directory to the location of your package.
3. In the Julia REPL, install `PkgTemplate.jl` as follows:
   
   ```
    using Pkg
    Pkg.add("PkgTemplates")
   ```